src/weights.rs

```diff
@@ -53,53 +53,53 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	// Storage: System Account (r:1 w:1)
 	// Storage: Assets Account (r:3 w:3)
 	fn create_exchange() -> Weight {
-		(103_019_000 as Weight)
-			.saturating_add(T::DbWeight::get().reads(7 as Weight))
-			.saturating_add(T::DbWeight::get().writes(7 as Weight))
+		Weight::from_ref_time(103_019_000 as u64)
```

Comment:
Good spot. I kept the `as u64` as that seemed to be the convention maintained within Substrate pallets (e.g. https://github.com/paritytech/substrate/blob/a3ed0119c45cdd0d571ad34e5b3ee7518c8cef8d/frame/assets/src/weights.rs#L111-L115) and wondered if it may have been part of the auto-generation process.

I'll remove as suggested for now.